README.md

```diff
@@ -85,5 +85,86 @@ iex> EchoClient.echo(pid, "Hi")
 *DBG* #PID<0.379.0> received frame: {:text, "Hi"}
 ```
 
+#### start_link/3 && start/3 
```

Comment:
I feel like this section would be better named **Supervision and Linking** with slightly altered content.

If you don't mind me writing this section. I think I understand the problem you had, but I would like to expand on and rephrase what you said.

Reply:
That's totally fine for me! I also think the Title suits better. Once there's enough content probably some links on top will help too. So should I just erase this part?

Reply:
That sounds perfect. I'll get something up by Friday.